src/addons/Portal/Portal.js

```diff
@@ -294,32 +318,41 @@ class Portal extends Component {
   }
 
   renderPortal() {
-    const { children, className } = this.props
+    const { children, className = '' } = this.props
 
     this.mountPortal()
 
     this.node.className = className
 
-    this.portal = ReactDOM.unstable_renderSubtreeIntoContainer(
+    ReactDOM.unstable_renderSubtreeIntoContainer(
       this,
       Children.only(children),
       this.node
     )
 
+    this.portal = this.node.firstElementChild
```

Comment:
```
LOG: 'children', Object{$$typeof: 60103, type: 'p', key: null, ref: null, props: Object{children: 'Hi'}, _owner: null, _store: Object{}}
LOG: 'firstElementChild', <p data-reactroot="">Hi</p>
      ✔ maintains ref to DOM node with host element
LOG: 'children', Object{$$typeof: 60103, type: function Component(props) { ... }, key: null, ref: null, props: Object{}, _owner: null, _store: Object{}}
LOG: 'firstElementChild', <p data-reactroot="">Hi</p>
      ✔ maintains ref to DOM node with React component
```

In both cases we need that actual DOM node, not a reference to the component or element.